pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/migration/StudyMigrator.java

```diff
@@ -38,4 +40,43 @@ public static void migrate(String studyName) {
         exportables.forEach(Exportable::export);
         log.info("Finished migration of DSM data to ES index {} for study {}", index, studyName);
     }
+
+    /**
+     * Export study data to ES for a list of participants
+     *
+     * @param exportLogs  list export logs to append to
+     */
+    public static void migrateParticipants(List<String> ddpParticipantIds, String studyName,
```

Comment:
Curious why we call "migrate" rather than just "export" 
We're just exporting DSM data into elastic and not really migrating from some other system or version.

Reply:
I'm not sure of the history, but I agree 'export' makes more sense. As is, we use the terms 'migrate' and 'export' interchangeably in code and comments.
However, there is related set of code in the `org.broadinstitute.dsm.export` package, referred to as 'export', which does some similar but different export stuff, apparently only as a RGP data fixup. Thus keeping 'migrate' for this area makes sense (even if the code history makes no sense).